### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#5469] [General]
@ckerr Hi Charles just looking at some typos that have come up here and there and wondered about this possibly being a false-positive. Should it be 'input'?

`./tests/libtransmission/variant-test.cc:180: inout ==> input, in out
./tests/libtransmission/variant-test.cc:181: inout ==> input, in out
./tests/libtransmission/variant-test.cc:183: inout ==> input, in out
./tests/libtransmission/variant-test.cc:186: inout ==> input, in out
./tests/libtransmission/variant-test.cc:187: inout ==> input, in out
./tests/libtransmission/variant-test.cc:191: inout ==> input, in out
./tests/libtransmission/variant-test.cc:194: inout ==> input, in out
./tests/libtransmission/variant-test.cc:195: inout ==> input, in out
./tests/libtransmission/variant-test.cc:197: inout ==> input, in out
./tests/libtransmission/variant-test.cc:200: inout ==> input, in out
./tests/libtransmission/variant-test.cc:201: inout ==> input, in out
./tests/libtransmission/variant-test.cc:205: inout ==> input, in out
./tests/libtransmission/variant-test.cc:208: inout ==> input, in out
./tests/libtransmission/variant-test.cc:209: inout ==> input, in out
./tests/libtransmission/variant-test.cc:213: inout ==> input, in out
`

**Comment by ckerr** [comment]
> @ckerr Hi Charles just looking at some typos that have come up here and there and wondered about this possibly being a false-positive. Should it be 'input'?

It's an [in-out](https://stackoverflow.com/questions/6900035/in-out-parameters-and-how-to-work-with-them-in-c) parameter, so the name is correct as-is :+1:
